aksel.nav.no/website/pages/templates/soknad-introside/intro-med-alternativ-soknad.tsx

```diff
@@ -0,0 +1,378 @@
+import { useEffect } from "react";
+import { ArrowRightIcon } from "@navikt/aksel-icons";
+import {
+  Accordion,
+  Bleed,
+  BodyLong,
+  BodyShort,
+  Box,
+  Button,
+  Checkbox,
+  GuidePanel,
+  Heading,
+  Link,
+  List,
+  Page,
+  Show,
+  Stack,
+  VStack,
+} from "@navikt/ds-react";
+
+function Example() {
+  useDekorator();
+
+  return (
+    <Page data-aksel-template="form-intropage-v1" footer={<Footer />}>
+      <Header />
+      <VStack as="main" gap="8">
+        <Page.Block width="text" gutters>
+          <Bleed marginInline={{ lg: "24" }}>
+            <Stack
+              align="center"
+              gap="6"
+              direction={{ xs: "row-reverse", lg: "row" }}
+              justify="start"
+              wrap={false}
+            >
+              <Show above="sm">
+                <ApplicationPictogram title="Application" />
+              </Show>
+              <VStack gap="1">
+                <BodyShort size="small">NAV 10-07.03 (om relevant)</BodyShort>
+                <Heading level="1" size="large">
+                  Søknad om [ytelse]
+                </Heading>
+              </VStack>
+            </Stack>
+          </Bleed>
+        </Page.Block>
+        <Page.Block width="text" gutters>
+          <VStack gap="12">
```

Comment:
I think this amount of gap is a little bit too much. I noticed it especially in the "alternativ-soknad" example. In Figma it's only 2rem. Is this something you have discussed?

Reply:
I think so, I'll double-check with Sjur.

Reply:
Reduced to gap="8" (2rem)